### PR TITLE
Bani connexml12

### PR DESCRIPTION
**Pull request by Bani2791**
1. Dynamic Dial: https://bani-connexml12--connexcs-docs.netlify.app/class5/connexml/#dynamic-dial
2. Press DTMF Variable: https://bani-connexml12--connexcs-docs.netlify.app/class5/connexml/#press-dtmf-variable
3. Before: https://bani-connexml12--connexcs-docs.netlify.app/class5/connexml/#before
4. Stream: https://bani-connexml12--connexcs-docs.netlify.app/class5/connexml/#stream
5. Echo: https://bani-connexml12--connexcs-docs.netlify.app/class5/connexml/#echo
6. Set: https://bani-connexml12--connexcs-docs.netlify.app/class5/connexml/#set
7. statuscallbackevent, statuscallback, statuscallbackmethod: https://bani-connexml12--connexcs-docs.netlify.app/class5/connexml/#dial

### CODE REVIEW

File: docs/class5/connexml.md [hunk 1]
```diff
@@ -484,6 +487,174 @@ An existing call is transferred to a different destination using the `Dial` ver
...
         </Response>
         ```
+    15. **statusCallbackEvent, statusCallback, statusCallbackMethod**
+        ``` xml
+        <?xml version="1.0" encoding="UTF-8"?>
+        <Response>
+            <Dial>
+                <Client
+                statusCallbackEvent="initiated ringing answered completed"
+                statusCallback="http://fr1js1.connexcs.net:3002"
```
Comment (AdamNawaz-connex) [inline docs/class5/connexml.md:497]: do not use fr1js1.connexcs.net add an example uri

File: docs/class5/connexml.md [hunk 2]
```diff
@@ -484,6 +487,174 @@ An existing call is transferred to a different destination using the `Dial` ver
...
+    <?xml version="1.0" encoding="UTF-8"?>
+    <Response>
+        <Before><Set name="newHome" value="Adam"/><Set name="headerName" value="Joe"/></Before>
+    </Response>
+    ```
+    Sets the variable `newHome` to "**Adam**".<br>Sets the variable `headerName` to "**Joe**"</br>.
+
+### Stream
+
+`Stream` is a noun together used with the `Start` verb.
```
Comment (AdamNawaz-connex) [inline docs/class5/connexml.md:575]: noun is used in conjunction with either <Start> or <Connect>.

File: docs/class5/connexml.md [hunk 3]
```diff
@@ -484,6 +487,174 @@ An existing call is transferred to a different destination using the `Dial` ver
...
+!!! Example
+    ``` xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <Response>
+        <Before><Set name="newHome" value="Adam"/><Set name="headerName" value="Joe"/></Before>
+    </Response>
+    ```
+    Sets the variable `newHome` to "**Adam**".<br>Sets the variable `headerName` to "**Joe**"</br>.
+
+### Stream
```
Comment (AdamNawaz-connex) [inline docs/class5/connexml.md:573]: add more details to stream like parameters
<Response>
	<Start>
		<Stream url="ws://fr1js1.connexcs.net:3001">
			<Parameter name="Hello" value="how are you, " />
		</Stream>
	</Start>

File: docs/class5/connexml.md [hunk 4]
```diff
@@ -484,6 +487,174 @@ An existing call is transferred to a different destination using the `Dial` ver
...
+    <?xml version="1.0" encoding="UTF-8"?>
+    <Response>
+        <Dial>
+            <Number>{% raw %}{{ substring Extension 9 }}{% endraw %}</Number>
+        </Dial>
+    </Response>
+    ```
+    The system will dial the number `"160"`
+
+### Press DTMF Variable
```
Comment (AdamNawaz-connex) [inline docs/class5/connexml.md:541]: Press

File: docs/class5/connexml.md (hunk 1 above)
Comment (Bani2791) [reply docs/class5/connexml.md:497]: corrected: statusCallback="https://api.example.com/compile?rateCompact=true&ids=101,202,303"
                statusCallbackMethod="POST">

File: docs/class5/connexml.md (hunk 3 above)
Comment (Bani2791) [reply docs/class5/connexml.md:573]: |**Attribute**|**Description**|**Options**|**Default Method**|
|-------------|---------------|-----------|-------|
|`url`|The WebSocket destination address for stream delivery||none|
|`name`(Optional)|Defines a unique custom name for the stream instance||none|
|`track`(Optional)|Indicates which track to stream|`inbound_track`, `outbound_track`, `both_tracks`|`inbound_track`|
|`statusCallback`|For every event listed in the `statusCallbackEvent` attribute, you may define a URL to send webhook requests to using the `statusCallback` attribute. A valid hostname is required for non-relative URLs (underscores are not allowed)|any `URL`|none|
|`statusCallbackMethod`|Lets you define the `HTTP` method ConnexCS should use when making requests to the URL specified in the `statusCallback` attribute.|`GET`, `POST`|`POST`|
|`bidirectionalMode`|Bidirectional streaming mode|`mp3`, `rtp`|`mp3`|
|`bidirectionalCodec`|Bidirectional streaming codec, used only with `bidirectionalMode=rtp`|`PCMU`, `PCMA`, `G722`|`PCMU`|

**Review by AdamNawaz-connex — APPROVED** [review]

**Pull request merged**